tests/test_web/test_api_v1.py

```diff
@@ -902,6 +902,7 @@ def test_patch_request_add_success(mock_smfsc, db, minimal_request_add, worker_a
         'state_reason': 'All done!',
         'index_image': 'index:image',
         'binary_image_resolved': 'binary-image@sha256:1234',
+        'distribution_scope': 'stage',
```

Comment:
`distribution_scope` is optional, right? Should not we have test for request when it is not set?

Reply:
good point. added! :)